src/components/DuosHeader.jsx

```diff
@@ -0,0 +1,537 @@
+  let currentUser = {};
+
+  if (isLogged) {
+    currentUser = Storage.getCurrentUser();
+  }
+
+  const contactUsSource = state.hover ? contactUsHover : contactUsStandard;
+  const contactUsIcon = isLogged ? '' : <img src={contactUsSource} style={{ display: 'inline-block', margin: '0 8px 0 0', verticalAlign: 'baseline' }} />;
+  const contactUsText = isLogged ? 'Contact Us' : <span style={{ display: 'inline', verticalAlign: 'text-bottom' }}>Contact Us</span>;
+  const contactUsButton = (
+    <button
+      id="btn_applyAcces"
+      style={{
+        color: state.hover ? '#2FA4E7' : '#ffffff',
+        fontSize: '14px',
+        fontWeight: '500',
+        background: 'transparent',
+        border: 'none',
+        outline: 'none',
+        margin: '0 20px 0 0',
+        padding: '15px 0',
+      }}
+      onMouseEnter={toggleHover}
+      onMouseLeave={toggleHover}
+      onClick={supportRequestModal}
+      data-tip="Need help? Contact us for some assistance"
+      data-for="tip_requestAccess"
+    >
+      {contactUsIcon}
+      {contactUsText}
+    </button>
+  );
+
+
+  const supportModal = (
+    <SupportRequestModal
+      showModal={state.showSupportRequestModal}
+      onOkRequest={okSupportRequestModal}
+      onCloseRequest={closeSupportRequestModal}
+      url={props.location.pathname}
+    />
+  );
+
+  const tabs = headerTabsConfig.filter((data) => data.isRendered(currentUser));
+
+  const isValidTab = (tab) => {
+    if (tab.link === location.pathname || location.pathname.includes(tab.search)) {
+      return true;
+    }
+    if (tab.children) {
+      return tab.children.some((subtab) => {
+        return subtab.link === location.pathname || location.pathname.includes(subtab.search);
+      });
+    }
+    return tab.children ? tab.children.indexOf((subtab) => subtab.link === location.pathname) !== -1 : false;
+  };
+
+  let initialSubTab = false;
+  let initialTab = false;
+
+  // note: location.state.selectedMenuTab will be populated if the user navigated
+  // to the current page by clicking on a tab from the nav bar.
+
+  // populate initialTab based on state (if valid) or by manually searching through all tabs.
+  if (location?.state?.selectedMenuTab && tabs.length > location.state.selectedMenuTab && isValidTab(tabs[location.state.selectedMenuTab])) {
+    initialTab = location.state.selectedMenuTab;
+  } else {
+    initialTab = tabs.findIndex(isValidTab);
+  }
+
+  if (initialTab !== -1) {
+    if (tabs[initialTab].link === location.pathname) {
+      initialSubTab = 0;
+    } else if (tabs[initialTab].children) {
+      initialSubTab = tabs[initialTab].children.findIndex((subtab) => {
+        return subtab.link === location.pathname || location.pathname.includes(subtab.search);
+      });
+    }
+  }
+
+  return (
+    <nav className="navbar-duos" role="navigation">
+      <Hidden mdDown={true}>
+        <div className="row no-margin" style={{ width: '100%' }}>
+          <NavigationTabsComponent
```

Comment:
This comment should appear between lines 463 & 464:
`          //Standard navbar for medium sized displays and higher (pre-existing navbar) `